iothub_client/inc/iothub_client_properties.h

```diff
@@ -58,51 +66,51 @@
 
 #include "iothub_client_core_common.h"
 
-/** @brief Current version of @p IOTHUB_CLIENT_REPORTED_PROPERTY structure.  */
-#define IOTHUB_CLIENT_REPORTED_PROPERTY_STRUCT_VERSION_1 1
+/** @brief Current version of @p IOTHUB_CLIENT_PROPERTY_REPORTED structure.  */
+#define IOTHUB_CLIENT_PROPERTY_REPORTED_STRUCT_VERSION_1 1
```

Comment:
are these public APIs?

Reply:
Yes - everything in iothub_client/inc/*.h is public.  The internal only is in the iothub_client/internal/* folder.